creator-node/src/dbManager.js

```diff
@@ -170,7 +170,7 @@ class DBManager {
    * Deletes all session tokens matching an Array of SessionTokens.
    *
    * @param {Array} sessionTokens from the SessionTokens table
-   * @param {Transaction} externalTransaction
+   * @param {Transaction=} externalTransaction
```

Comment:
is this a typo or some syntax I'm not familiar with?

Reply:
I believe that's how you make a type optional in jsDocs

Reply:
synced offline - was only aware of the `[]` way but looks like both are equally valid so good to go from my end 👍